DESCRIPTION

```diff
@@ -29,7 +29,7 @@ Imports:
 Suggests:
     DT,
     gsm.kri (>= 1.1.1),
-    gsm.mapping (>= 1.0.2),
+    gsm.mapping (>= 1.0.1),
```

Comment:
```suggestion
    gsm.mapping (>= 1.1.0),
```

Just to ensure the appropriate `VISIT` related stuff works, it affects the vignette